core/bentley/src/Logger.ts

```diff
@@ -246,6 +246,12 @@ export class Logger {
   }
 
   private static getExceptionMessage(err: unknown): string {
+    if (err === undefined) {
```

Comment:
If we think it's valuable to convey the error was specifically null or undefined this is cool. But if not, it's an opportunity for the only good use of JS loose equality:

`err == null` will cover both undefined and null.

Reply:
Note that the exact string used here also shows up in the updated unit test.

Reply:
Yeah I can see that. Sounds good,